templates/base/head_script.tmpl

```diff
@@ -0,0 +1,44 @@
+<script>
+	<!-- /* eslint-disable */ -->
+	window.addEventListener('error', function(e) {window._globalHandlerErrors=window._globalHandlerErrors||[]; window._globalHandlerErrors.push(e);});
```

Comment:
Should probably create a separate webpack chunk that loads before`index.js` for this instead of inlining it. That way, you also shouldn't need this global variable.

Reply:
However, `index.js` is very late, the layout is:

```
window.addEventListener('error',)
window.config = 
template <script>
index.js
```

So the `window.addEventListener` must be the first one in script tag.